frontend/src/reducer/notificationList.js

```diff
@@ -1,15 +1,21 @@
 import { uniqBy } from 'lodash'
 import {
   ADD,
+  APPEND,
+  NEXT_PAGE,
   NOTIFICATION,
   NOTIFICATION_LIST,
   SET,
   UPDATE
 } from '../action-creator.sync.js'
 
-const defaultNotificationList = []
+const defaultNotificationsObject = {
+  list: [],
+  hasNextPage: false,
+  nextPageToken: ''
+}
 
-export default function notificationList (state = defaultNotificationList, action) {
+export default function notificationList (state = defaultNotificationsObject, action) {
```

Comment:
I have issue with the `List` in the reducer's name since its default state is an object

Reply:
Okay, I kept the name because the other properties are related to the list, the list is the main element of these object. I don't love notificationsObject because I feel that it refers to various objects, one for each notification , maybe `notificationListObject`? Or do you still prefer `notificationsObject`?

Reply:
Changed to `notificationPage`